cfme/configure/settings.py

```diff
@@ -1,50 +1,43 @@
-# -*- coding: utf-8 -*-
-
-""" Module dealing with Configure/My Setting section."""
-
-from functools import partial
 import re
-
-import cfme.fixtures.pytest_selenium as sel
-import cfme.web_ui.tabstrip as tabs
-import cfme.web_ui.toolbar as tb
-from cfme.web_ui import (
-    AngularSelect, Form, Region, fill, form_buttons, flash, Table, ButtonGroup, Quadicon,
-    CheckboxTree, Input, CFMECheckbox, BootstrapTreeview, match_location)
-from navmazing import NavigateToSibling, NavigateToAttribute
-from utils import version, deferred_verpick
+from navmazing import NavigateToAttribute
+from widgetastic_manageiq import Table, BootstrapSelect, BreadCrumb, Text, ViewButtonGroup
+from widgetastic_patternfly import (BootstrapSwitch,
+                                    Input, Button, CheckableBootstrapTreeview)
+from widgetastic.utils import VersionPick, Version
+from widgetastic.widget import View
+from cfme.base.ui import MySettingsView
 from utils.pretty import Pretty
 from utils.update import Updateable
 from utils.appliance import Navigatable
 from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
 
 
-details_page = Region(infoblock_type='detail')
+class TimeProfileAddForm(MySettingsView):
```

Comment:
**REQUIRED**
I don't think it's correct to inherit `MySettingsView` here. In fact each tab of `MySettingsView` should have respectful nested view or maybe even better to have nested `MySettingsView` in each view.